packages/core/src/data-fetching/api/PrezlyApi.ts

```diff
@@ -80,7 +76,9 @@ export class PrezlyApi {
         }
 
         try {
-            const story = await this.sdk.stories.get(uuid);
+            const story = await this.sdk.stories.get(uuid, {
+                formats: [Story.FormatVersion.SLATEJS_V4],
```

Comment:
I wasn't sure if I should specify the V4 format in `searchStories` as well? 🤔 Since those are mainly used for listing and such so you're probably not gonna be working with `content`, but I wasn't sure.

Also not sure what the effect on backend would be in this case, when you request a lot of stories and also want them converted to V4 🤔 

Maybe @e1himself or @pestaa can advise here.

Reply:
It's better use the same version for all requests (for completeness). Also, some themes may decide to use `content` on the listing as well (to render `titles` with bold/italic formatting, for example). The performance impact should be negligible, so don't worry about it.

Reply:
Aight 👍

Reply:
Yep, we do the same in legacy themes. No need to watch out what's going on downstream.